src/modules/logger/logger.cpp

```diff
@@ -575,53 +575,60 @@ void Logger::add_common_topics()
 	// Note: try to avoid setting the interval where possible, as it increases RAM usage
 	add_topic("actuator_controls_0", 100);
 	add_topic("actuator_controls_1", 100);
-	add_topic("actuator_outputs", 100);
-	add_topic("airspeed");
+	add_topic("actuator_outputs", 200);
+	add_topic("airspeed", 200);
 	add_topic("att_pos_mocap", 50);
-	add_topic("battery_status", 300);
+	add_topic("battery_status", 500);
 	add_topic("camera_capture");
 	add_topic("camera_trigger");
-	add_topic("commander_state", 100);
+	add_topic("commander_state", 200);
 	add_topic("control_state", 100);
 	add_topic("cpuload");
-	add_topic("differential_pressure", 50);
-	add_topic("distance_sensor");
-	add_topic("ekf2_innovations", 50);
-	add_topic("esc_status", 250);
-	add_topic("estimator_status", 200); //this one is large
-	add_topic("gps_dump"); //this will only be published if gps_dump_comm is set
```

Comment:
I wouldn't comment it either.
It can be used to dump the raw GPS device communication. This can then be used for further analysis. I used it for RTK, the dump can be opened with the u-blox tool u-center.
We can conditionally add the topic if the param `GPS_DUMP_COMM` is set (requires a reboot anyway).